packages/shared/models/Court.js

```diff
@@ -37,9 +37,9 @@ module.exports = class {
 
   async registry() {
     if (!this._registry) {
-      const address = await this.instance.getJurorsRegistry()
-      const JurorsRegistry = await this.environment.getArtifact('JurorsRegistry', '@aragon/court')
-      this._registry = await JurorsRegistry.at(address)
+      const address = await this.instance.getGuardiansRegistry()
+      const GuardiansRegistry = await this.environment.getArtifact('GuardiansRegistry', '@aragon/court')
```

Comment:
Is this already available? Do you also want to update `@aragon/court` ?

Reply:
Yes, but I updated the dependency in a further PR